src/wp-content/themes/twentytwentyfour/patterns/hidden-sidebar.php

```diff
@@ -6,69 +6,66 @@
 <h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'About the author', 'twentytwentyfour' ); ?></h2>
 <!-- /wp:heading -->
 
 <!-- wp:post-author-biography {"fontSize":"small"} /--></div>
+<!-- /wp:group --></div>
 <!-- /wp:group -->
 
 <!-- wp:separator {"backgroundColor":"base-3","className":"is-style-wide"} -->
 <hr class="wp-block-separator has-text-color has-base-3-color has-alpha-channel-opacity has-base-3-background-color has-background is-style-wide"/>
 <!-- /wp:separator -->
 
-<!-- wp:group {"layout":{"type":"constrained"}} -->
-<div class="wp-block-group">
-	<!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
-	<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'Latest Posts', 'twentytwentyfour' ); ?></h2>
-	<!-- /wp:heading -->
+<!-- wp:group {"style":{"spacing":{"blockGap":"16px"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
+<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'Popular Categories', 'twentytwentyfour' ); ?></h2>
+<!-- /wp:heading -->
 
-	<!-- wp:query {"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false}} -->
-	<div class="wp-block-query"><!-- wp:post-template -->
-	<!-- wp:group {"style":{"spacing":{"blockGap":"2px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
-	<div class="wp-block-group"><!-- wp:post-title {"isLink":true,"fontSize":"medium"} /-->
-	<!-- wp:template-part {"slug":"post-meta","theme":"twentytwentyfour"} /-->
-	</div>
-	<!-- /wp:group -->
+<!-- wp:categories {"showHierarchy":true,"showPostCounts":true,"fontSize":"small"} /--></div>
+<!-- /wp:group -->
 
-	<!-- wp:spacer {"height":"var:preset|spacing|10"} -->
-	<div style="height:var(--wp--preset--spacing--10)" aria-hidden="true" class="wp-block-spacer"></div>
-	<!-- /wp:spacer -->
-	<!-- /wp:post-template --></div>
-	<!-- /wp:query -->
+<!-- wp:separator {"backgroundColor":"base-3","className":"is-style-wide"} -->
+<hr class="wp-block-separator has-text-color has-base-3-color has-alpha-channel-opacity has-base-3-background-color has-background is-style-wide"/>
+<!-- /wp:separator -->
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"26px"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"16px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-group"><!-- wp:heading {"style":{"typography":{"fontSize":"1.6rem"}}} -->
+<h2 class="wp-block-heading" style="font-size:1.6rem"><?php echo esc_html__( 'Useful Links', 'twentytwentyfour' ); ?></h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size"><?php echo esc_html__( 'Links I found useful and wanted to share.', 'twentytwentyfour' ); ?></p>
```

Comment:
```suggestion
<p class="has-small-font-size"><?php esc_html_e( 'Links I found useful and wanted to share.', 'twentytwentyfour' ); ?></p>
```

There's a few of these in the PR.

Reply:
I think I replaced all instances of this

Reply:
Looks good! Thank you!